### ISSUE TITLE
Clarification of "Lesson" vs "Episode" in the text

### ISSUE THREAD

**Issue by dvalters** [swcarpentry/python-novice-inflammation#602]
In the text of some of the episodes, the words _lesson_ and _episode_ are used interchangeably in some places, which lead to a bit of confusion for in preparing for the instructor training. I think it is clear now (Lesson - a whole set of episodes on a core topic i.e. Python, Unix Shell etc.; Episode - each subsection/separate page with a number in the Lesson)

I was going to open a PR with some minor changes to fix this, if that seems reasonable?

**Comment by maxim-belkin** [comment]
Absolutely! Please do so, changing one episode per PR.

**Comment by maxim-belkin** [comment]
Thanks for your pull requests! Have you found any other improper uses of the word `lesson`?

**Comment by dvalters** [comment]
Yep - couple more PR's to follow - just a bit caught up with other work today
